counter/package.json

```diff
@@ -6,6 +6,6 @@
   "dependencies": {
```

Comment:
god bless you for listening to my advice, and my suggestion to name it junokit !
well done, this will make life easier